### PR TITLE
[MXNET-48] update on setting up Scala with MXNet and the IntelliJ IDE

### LINKED ISSUE #9973: update Scala tutorial - using IntelliJ IDE
## Description
http://mxnet.incubator.apache.org/tutorials/scala/mxnet_scala_on_intellij.html

1. Dated info - update and verify actual setup
2. Broken link - "this procedure" link for mxnet package for scala setup. Current instructions are here: http://mxnet.incubator.apache.org/install/osx_setup.html
3. Scala plugin setup for IDE link - this is old. Switch to https://www.jetbrains.com/help/idea/scala.html

## Comments
I plan to address these issues.

### PR DESCRIPTION
**Pull request by aaronmarkham**
## Description ##
Fixes #9973
Documentation update for MXNet-Scala setup and setting up an IntelliJ project.

## Comments ##

Improvements needed on another pass:
* Command line `mvn clean package` will show build issues while IDE's "Rebuild" does not, so some attention should be paid to this.
* Paths and variables in the .pom should be better implemented.
* A hello world should be appended to this tutorial, so we can show that MXNet is working via the Scala jar file.

### CODE REVIEW

**Comment by piiswrong** [comment]
Please move pics to https://github.com/dmlc/web-data/pulls

**Comment by aaronmarkham** [comment]
Images PR: https://github.com/dmlc/web-data/pull/42

**Comment by CodingCat** [comment]
Hi, the community has passed to vote about associating the code changes with JIRA (https://lists.apache.org/thread.html/ab22cf0e35f1bce2c3bf3bec2bc5b85a9583a3fe7fd56ba1bbade55f@%3Cdev.mxnet.apache.org%3E)

We have updated the guidelines for contributors in https://cwiki.apache.org/confluence/display/MXNET/Development+Process, please ensure that you have created a JIRA at https://issues.apache.org/jira/projects/MXNET/issues/ to describe your work in this pull request and include the JIRA title in your PR as [MXNET-xxxx] your title where MXNET-xxxx is the jira id

**Comment by aaronmarkham** [comment]
https://issues.apache.org/jira/projects/MXNET/issues/MXNET-48

File: docs/tutorials/scala/mxnet_scala_on_intellij.md [hunk 1]
```diff
@@ -1,62 +1,295 @@
...
 
 ## Prerequisites:
-To use this tutorial, you need:
+To use this tutorial you need the following items, however after this list, installation info for macOS is provided for your benefit:
+
+- [Java 8 JDK](http://www.oracle.com/technetwork/java/javase/downloads/index.html)
+- [Maven](https://maven.apache.org/install.html)
+- [Scala](https://www.scala-lang.org/download/) - comes with IntelliJ, so you don't need to install it separately
+- [MXNet Shared Library and Scala Package](#build-the-mxnet-shared-library-and-scala-package)
+- [IntelliJ IDE](https://www.jetbrains.com/idea/)
```
Comment (eric-haibin-lin) [inline docs/tutorials/scala/mxnet_scala_on_intellij.md]: Do we really want to have IDEs like IntelliJ as part of the prerequisite?

Comment (aaronmarkham) [reply docs/tutorials/scala/mxnet_scala_on_intellij.md]: Fixed the conflict...

Well this is purpose of the tutorial... to setup intellij.
The scala install page won't have this requirement... just an option.

**Comment by aaronmarkham** [comment]
@eric-haibin-lin @nswamy - resolved the conflict.

**Pull request merged**